src/preprocessor.ts

```diff
@@ -12,21 +13,29 @@ export function process(
   config: JestConfig,
   transformOptions: TransformOptions = { instrument: false },
 ) {
+  enableLoggingIfNeeded(config);
```

Comment:
this line moves the comment below which makes the comment lose some context

Reply:
Nicely spotted. I'll move it.